src/main/configurations/Translate/Configuration_PatchRelevanteAndereZaakDelete.xml

```diff
@@ -0,0 +1,108 @@
+                <Forward name="else" path="LogWarningZaakNotRelated" />
+            </XmlIfPipe>
+
+            <SenderPipe
+                name="LogWarningZaakNotRelated"
+                >
+                <LogSender
+                    name="LogWarningSender" />
+                <Forward name="success" path="EXIT" />
+            </SenderPipe>
+
+            <XsltPipe name="CreatePatchZaakBodyToDeleteAndereZaak"
+                getInputFromSessionKey="originalMessage"
+                styleSheetName="CreeerZaak_LK01/xsl/CreateZgwAndereZaakBodyToDelete.xslt"
+                >
+                <Param name="AndereUrlToDelete" sessionKey="AndereUrlToDelete"/>
+                <Param name="AndereUrlsToDelete" sessionKey="AndereUrlsToDelete" type="DOMDOC"/>
+                <Forward name="success" path="CheckIfRelevanteAndereZakenListEmpty"/>
+            </XsltPipe>
+
+            <XmlIfPipe name="CheckIfRelevanteAndereZakenListEmpty"
+                xpathExpression="/container/relevanteAndereZaken[1]/url">
+                <Forward name="then" path="XmlToJson"/>
+                <Forward name="else" path="CreateEmptyRelevanteAndereZakenJson"/>
+            </XmlIfPipe>
+
+            <XsltPipe name="CreateEmptyRelevanteAndereZakenJson"
+                styleSheetName="UpdateZaak_LK01/xsl/CreateEmptyRelevanteAndereZakenJson.xslt">
+                <Forward name="success" path="PatchZaak" />
+            </XsltPipe>
+
+            <Json2XmlValidatorPipe
+                name="XmlToJson"
+                root="container"
+                schema="CreeerZaak_LK01/xsd/AndereZaakBodyToJson.xsd"
+                outputFormat="JSON"
+                compactJsonArrays="false"
+                throwException="true">
+                <Forward name="success" path="PatchZaak" />
+            </Json2XmlValidatorPipe>
+
+            <SenderPipe name="PatchZaak">
+				<HttpSender 
+					name="PatchZaakSender" 
+					methodType="PATCH"
+					headersParams="Authorization,Accept-Crs,Content-Crs"
+                    contentType="application/json"
+                    timeout="${zaakbrug.zgw.zaken-api.timeout}"
+                    maxExecuteRetries="5"
+				/>
+                <Param name="url" sessionKey="Url"/>
+                <Param name="Accept-Crs" value="EPSG:4326"/>
+                <Param name="Content-Crs"  value="EPSG:4326"/>
+                <Param name="Authorization"  sessionKey="Authorization" />
+                <Forward name="success" path="EXIT" />
+                <Forward name="exception" path="ErrorJsonToXml" />
+			</SenderPipe>
+            <JsonPipe name="ErrorJsonToXml">
+                <Forward name="success" path="buildErrorMsg" />
+            </JsonPipe>
+            <XsltPipe name="buildErrorMsg"
+                styleSheetName="Common/xsl/ParseNegativeHttpResult.xsl">
+                <Param name="senderPipeName" value="PatchZaak" />
+                <Forward name="success" path="EXCEPTION" />
+            </XsltPipe>
```

Comment:
This + `GetAuthorizationHeaderForZakenApi` at the start of this adapter should be replaced with a call to the already existing `Zaken_UpdateZgwZaak`. Just need to make sure the root element is called `ZgwZaak` instead of `container`.